Add note to update pins as part of mpl-sphinx-theme release (#57)
matplotlib/mpl-sphinx-theme: README.rst

   $ python -m build -s -w
   $ twine upload dist/mpl_sphinx_theme-x.y.z*

> Update the requirements for the following sites which use ``mpl-sphinx-theme``:

Reviewer: More specific
```suggestion
Update the required ``mpl-sphinx-theme`` version in the following files:
```
Alternative: "Update the pinned ..."
